Extensions/PlatformBehavior/platformerobjectruntimebehavior.js

```diff
@@ -861,6 +862,14 @@ gdjs.PlatformerObjectRuntimeBehavior.prototype.getMaxFallingSpeed = function() {
   return this._maxFallingSpeed;
 };
 
+/**
+ * Get the maximum speed of Climbing the ladder
```

Comment:
This is a speed, not a maximum speed. Same for the other descriptions

Reply:
I corrected the mistake, Thanks :)